app/Makefile

```diff
@@ -171,11 +176,21 @@ lint-security: # https://bandit.readthedocs.io/en/latest/index.html
 
 
 ##################################################
-# Scripts
+# CLI Commands
 ##################################################
 
-create-user-csv:
-	$(PY_RUN_CMD) create-user-csv
+cmd: ## Run Flask app CLI command (Run `make cli args="--help"` to see list of CLI commands)
+	$(FLASK_CMD) $(args)
+
+cmd-user-create-csv: ## Create a CSV of the useres in the database (Run `make cli-user-create-csv args="--help"` to see the command's options)
+	$(FLASK_CMD) user create-csv $(args)
+
+# Set init-db as pre-requisite since there seems to be a race condition
+# where the DB can't yet receive connections if it's starting from a 
+# clean state (e.g. after make stop, make clean-volumes, make openapi-spec)
+openapi-spec: init-db ## Generate OpenAPI spec
+	$(FLASK_CMD) spec --format yaml --output ./openapi.yml
```

Comment:
I had to deal with that race condition on NJ as the Oracle DB takes 60+ seconds to startup: https://github.com/navapbc/archive-nj-dol-ui-claimant-api/blob/main/wait-for-local-oracle.sh - it just checks if a particular log message appears every 5 seconds.

Reply:
Oh interesting! I wonder if there's a better way to do that here as well.